tools/docker-format/Dockerfile

```diff
@@ -1,8 +1,28 @@
 FROM ubuntu:cosmic
 
+# repositories are obsolete. the image need to be changed.
+RUN perl -pi -e 's@//archive@//old-releases@g' /etc/apt/sources.list
+# disable security updates altogether
+RUN perl -pi -e 's@^(.*security.*)$@#$1@g' /etc/apt/sources.list
+
 RUN apt update && \
-    apt install -y clang-format golang git python-pip && \
-    go get -v github.com/bazelbuild/buildtools/buildifier && \
+    apt install -y clang-format git python-pip && \
     pip install 'cmake_format>=0.5.2'
 
+# brings golang to 1.10 by default.bazelbuilds wants 1.12 minimum....
+#RUN apt install -y golang
+
+RUN apt install -y curl
```

Comment:
Please merge all of the RUNs together into a single one to avoid a long chain of docker overlays.